modules/conflict_resolver/php/resolved_conflicts.class.inc

```diff
@@ -197,7 +196,6 @@ class Resolved_Conflicts extends \NDB_Menu_Filter
         // Project list, if applicable
         if ($config->getSetting('useProjects')==='true') {
             $projectList      = \Utility::getProjectList();
-            $list_of_projects = array('' => 'All') + $projectList;
             foreach ($projectList as $key => $value) {
                 $list_of_projects[$value] = $value;
```

Comment:
same here? not sure if this is maybe done way above

Reply:
@zaliqarosli thanks!